migrate/vars.go

```diff
@@ -0,0 +1,43 @@
+package migrate
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
+	errFileNamingTpl      = "bad file name '%s', must be like '<timestamp>_something.<up|down>.sql'"
+	errFileVersionTpl     = "bad file version '%s', must be greater than 0"
+	errVersionNotEqualTpl = "version of 'up' and 'down' migrations must be equal: %d != %d"
+
+	fileNameTpl = "%d_%s.%s.sql"
+
+	sqlSelectVersion = `SELECT version, created_at FROM ? ORDER BY id ASC`
+	sqlCreateSchema  = `CREATE SCHEMA IF NOT EXISTS ?`
+	sqlNewVersion    = `INSERT INTO ? (version, created_at) VALUES (?, now())`
+	sqlRemVersion    = `DELETE FROM ? WHERE version = ?`
+	sqlGetVersion    = `SELECT version FROM ? ORDER BY id DESC LIMIT 1`
+	sqlCreateTable   = `
+CREATE TABLE IF NOT EXISTS ? (
+	id serial,
+	version bigint UNIQUE,
```

Comment:
maybe we should add full migration name?

Reply:
fixed